lib/rubocop/cop/rspec/sort_tags.rb

```diff
@@ -0,0 +1,333 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Sort RSpec tags alphabetically.
+      #
+      # @example EnforcedStyle: sort_only (default)
+      #   # bad
+      #   describe 'Something', :b, :a
+      #   context 'Something', foo: 'bar', baz: true
+      #   it 'works', :b, :a, foo: 'bar', baz: true
+      #
+      #   # good
+      #   describe 'Something', :a, :b
+      #   context 'Something', baz: true, foo: 'bar'
+      #   it 'works', :a, :b, baz: true, foo: 'bar'
+      #
+      # @example EnforcedStyle: hash
+      #   # bad
+      #   describe 'Something', :b, :a
+      #   context 'Something', foo: 'bar', baz: true
+      #   it 'works', :b, :a, foo: 'bar', baz: true
+      #
+      #   # good
+      #   describe 'Something', a: true, b: true
+      #   context 'Something', baz: true, foo: 'bar'
+      #   it 'works', a: true, b: true, baz: true, foo: 'bar'
+      #
+      # @example EnforcedStyle: prefer_symbols
+      #   # bad
+      #   describe 'Something', :b, :a
+      #   context 'Something', foo: 'bar', baz: true
+      #   it 'works', :b, :a, foo: 'bar', baz: true
+      #
+      #   # good
+      #   describe 'Something', :a, :b
+      #   context 'Something', :baz, foo: 'bar'
+      #   it 'works', :a, :b, :baz, foo: 'bar'
+      #
+      class SortTags < Base
```

Comment:
The first thing that strikes me is the term "tags". RSpec usually refers to it as "metadata". There are mentions of "tag", too, and there's even a [doc page named after "tag"](https://relishapp.com/rspec/rspec-core/v/3-11/docs/command-line/tag-option), I believe ["metadata"](https://relishapp.com/rspec/rspec-core/v/3-11/docs/metadata/user-defined-metadata) is used more often.